runtime/types.go

```diff
@@ -48,7 +48,7 @@ const (
 	// Supported signing algorithms
 	SignatureAlgorithmUnknown         = sema.SignatureAlgorithmUnknown
 	SignatureAlgorithmECDSA_P256      = sema.SignatureAlgorithmECDSA_P256
-	SignatureAlgorithmECDSA_Secp256k1 = sema.SignatureAlgorithmECDSA_Secp256k1
+	SignatureAlgorithmECDSA_Secp256k1 = sema.SignatureAlgorithmECDSA_secp256k1
```

Comment:
I didn't change the interface type (this constant) so that this change won't affect FVM.

Reply:
We should change it everywhere to be consistent here, it's also upper-case on the FVM side